spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Sequential.scala

```diff
@@ -32,6 +33,37 @@ import scala.collection.mutable.ArrayBuffer
 class Sequential[T: ClassTag]
 (implicit ev: TensorNumeric[T]) extends Container[Activity, Activity, T] {
 
+  private[bigdl] override def compilingPath(): List[Node[AbstractModule[Activity, Activity, T]]] = {
```

Comment:
Maybe we needn't a compileingPath method. There's a toGraph method, we can convert the container to a graph then do the shape computing in the graph